Tests/Runtime/Attributes/BuildSceneAttributeInferredTest.cs

```diff
@@ -0,0 +1,30 @@
+// Copyright (c) 2023-2024 Koji Hasegawa.
+// This software is released under the MIT License.
+
+using System.Threading.Tasks;
+using Cysharp.Threading.Tasks;
+using NUnit.Framework;
+using TestHelper.RuntimeInternals;
+using UnityEngine;
+
+namespace TestHelper.Attributes
+{
+    /// <seealso cref="LoadSceneAttributeTest"/>
+    [TestFixture]
+    [BuildScene]
+    public class BuildSceneAttributeInferredTest
```

Comment:
Please add a method to the BuildSceneAttributeTest class. (Do not create this class.)
BuildSceneAttributes can be multiple specified.